biz.aQute.junit.platform/readme.md

```diff
@@ -0,0 +1,6 @@
+# JUnit Platform Tester
+
+This is a `Tester-Plugin` implementation that uses the JUnit Platform library \
+	to discover and launch tests. It is mostly compatible with `biz.aQute.tester` \
+	and (like `biz.aQute.tester`) adds itself to `-runbundles`, rather than \
+	adding itself to `-runpath` (like `biz.aQute.junit`).
```

Comment:
The naming can be confusing here. We currently have biz.aQute.junit and biz.aQute.tester. This new bundle is more like biz.aQute.tester (since it does not contain the junit packages and is a Tester-Plugin) but its name is like biz.aQute.junit which could imply it is like biz.aQute.junit and contains the junit packages.

I wonder if a better name might be biz.aQute.tester.jupiter? This would make the similarity with biz.aQute.tester more clear and also make the affinity with junit jupiter also explicit?

Reply:
I agree this is confusing, and I'm very open to alternative suggestions. However, I don't think that any name that includes "Jupiter" is appropriate, as I will explain.

Part of the reason for the confusion comes from the architectural change in JUnit 5. JUnit 3 & 4 were monolithic jars, but JUnit 5 has become more modular. From the [JUnit 5 docs](https://junit.org/junit5/docs/current/user-guide/#overview-what-is-junit-5), "JUnit 5 = JUnit Platform + JUnit Jupiter + JUnit Vintage".

* **JUnit Platform** refers to the test discovery & launching framework. It does not implement any particular type of testing, but defines common interfaces that test environments need. One of these is the `TestEngine` interface, concrete implementations of which comprise the actual testers.
* **JUnit Jupiter** refers to all of the packages in the `org.junit.jupiter.*` namespace, which includes all of the Jupiter assertions, annotations, and the Jupiter `TestEngine` implementation itself.
* **JUnit Vintage** is an implementation of the `TestEngine` interface that executes JUnit 3 & 4 tests.

In the future it is possible that other JVM-based test environments (like Spock, TestNG, or JUnit Saturn) could get their own `TestEngine` implementations, and they would plug in to the same JUnit Platform launcher environment. They should also (if appropriately bundled) simply plug-and-play with this bundle.

So hopefully from the above, you'll appreciate that while I'm open to alternative suggestions, I firmly believe that Jupiter is terminologically incorrect for this bundle, and if we include "jupiter" anywhere in this bundle's name it will further compound the confusion. That is why I settled on a name that included "JUnit Platform" in it, because I thought in the long run it would help to overcome this confusion.

Other variations I played with that make the similarity with `biz.aQute.tester` more clear:
* `biz.aQute.tester.junit.platform`
* `biz.aQute.tester.junit-platform` - more tightly couples the "JUnit" and "Platform" words together.

I eventually ditched these in favour of what I have now simply because I thought the above were too long. However, based on your feedback, perhaps `biz.aQute.tester.junit-platform` would be a good candidate - making the similarity with `biz.aQute.tester` more clear and the affinity with JUnit Platform more explicit?

Reply:
Yes, I agree `biz.aQute.tester.junit-platform` is a better name. Please make this change and related changes to associated packages, etc.

Reply:
Done.